ndcube/ndcube.py

```diff
@@ -258,33 +235,32 @@ def world_to_pixel(self, *quantity_axis_list):
         result = [u.Quantity(world_to_pixel[index], unit=u.pix) for index in range(self.wcs.pixel_n_dim)]
         return result[::-1]
 
-    def axis_world_coords(self, *axes, edges=False):
+    def axis_world_coords(self, axes=(), edges=False, skycoord=True):
```

Comment:
why the change from `*axes`?

Reply:
I was getting an error of using keyword arguments after `*axes`.